app/services/idv/steps/document_capture_step.rb

```diff
@@ -105,7 +105,9 @@ def handle_stored_result
       end
 
       def stored_result
-        @stored_document_capture_session_result ||= document_capture_session&.load_result
+        return @stored_result if defined?(@stored_result)
+        @stored_result = document_capture_session&.load_result ||
```

Comment:
is there a reason you are doing that return instead of maybe doing `@stored_result ||=  document_capture_session&.load_result || document_capture_session&.load_doc_auth_async_result`

Reply:
> is there a reason you are doing that return instead of maybe doing `@stored_result ||= document_capture_session&.load_result || document_capture_session&.load_doc_auth_async_result`

One or both of the results could be `nil`. Using `defined?` makes it so that we'll return the memoized `nil` value. `||=` would cause it to try to load again every time the method is called.